CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## Unreleased
+### Fixed
+- The Conjur version is now printed on server startup, after running `conjurctl server`
+  ((cyberark/conjur#1590)[https://github.com/cyberark/conjur/pull/1590])
```

Comment:
The parens are backwards here. The issue should have square brackets and the link should have parens.